data/reducers/frames.js

```diff
@@ -90,25 +90,22 @@ function addFrames(state, newFrames) {
   });
 
   // Apply filter on incoming frames.
-  if (newState.filter.text) {
-    return filterFrames(newState, newState.filter);
-  }
-
-  return newState;
+  return filterFrames(newState, newState.filter);
```

Comment:
This fixed an issue of new frames not showing up when switching back to "No socket ID filter". I was not able to see any adverse effects of doing this, so I left it in.